python/utils/bootstrap_utilities.py

```diff
@@ -278,6 +278,13 @@ def start_app(engine):
         return 0
 
 
+def _exc_to_str(exc):
+    if sys.version_info[0] >= 0:
```

Comment:
when would this not be greater or equal to 0?
This is the major version of Python, is there some weird situation where this is not the case?

Reply:
WHAT HAVE I DONE!